ui/app/serializers/node-attributes.js

```diff
@@ -2,6 +2,6 @@ import ApplicationSerializer from './application';
 
 export default ApplicationSerializer.extend({
   normalize(typeHash, hash) {
-    return this._super(typeHash, { Attributes: hash });
+    return this._super(typeHash, { NodeAttributes: hash });
```

Comment:
`attributes` seems to have become somewhat of a reserved word, so this gives it a redundant-but-functional new name.

Reply:
Ah, my least favorite part about Ember.